### ISSUE TITLE
ARM (M1 Mac) support is missing

### ISSUE THREAD

**Issue by exoego** [treasure-data/lambda-local-proxy#2]
# Error happened
When I started working on Treasure Function on M1 Mac, I found that functions can not be executed locally.
It shows error like this

```
❯ ext/start-api
Error response from daemon: OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec format error: unknown
```

# Observation
`function-template-***` for Treasure Functions refer binary of `lambda-local-proxy` in `ext/start-api`.
https://github.com/treasure-data/provisioner-function-template/blob/b38b4ab72d57d43bf5e4d0bbb705e0d771745234/ext/start-api#L16

It downloads `_Linux_x86_64.tar.gz`, so I assume it does not run on ARM (Apple Silicon)-based M1 Mac.

# Expectation
Since Apple is moving to ARM instead of Intel, many developers will eventually switches to ARM Mac, at least in Treasure Data.
So, I think ARM support is essential.

**Comment by exoego** [comment]
I will work on this.

**Comment by exoego** [comment]
Fixed in v0.0.5
